### PR TITLE
fix: fix wrong prop types and errors in storybook

### PR DESCRIPTION
**Pull request by satazor**
This PR has two commits, one of them updates storybook to v5.

When merging, please use the "Rebase and Merge"

### CODE REVIEW

File: src/components/feedback-message/README.md [hunk 1]
```diff
@@ -20,13 +20,12 @@ import { FeedbackMessage } from '@nomios/web-uikit';
 
 | name | type | default | description |
 | ---- | ---- | ------- | ----------- |
-| children | string | *required* | Text to render. |
+| children | node | *required* | The contents to render, usually a text |
```
Comment (PedroMiguelSS) [inline src/components/feedback-message/README.md]: It seems weird to me that a `feedback-message` component can get other than text as children. But I think I got your point.

**Review by dominguesgm — APPROVED** [review]
Everything looks good to me 👍

File: src/components/feedback-message/README.md (hunk 1 above)
Comment (PedroMiguelSS) [inline src/components/feedback-message/README.md:23]: It seems weird to me that a `feedback-message` component can get other than text as children. But I think I got your point.

**Review by PedroMiguelSS — APPROVED** [review]
Looks great! Sorry about some senseless errors! 🤦‍♂️

File: src/components/feedback-message/README.md (hunk 1 above)
Comment (satazor) [reply src/components/feedback-message/README.md:23]: We can change to `string` in the FeedbackMessage proptypes. Either works.

Comment (PedroMiguelSS) [reply src/components/feedback-message/README.md:23]: Don't think it's really necessary. It's more generic like this. For me, we can merge as it is!

**Pull request merged**